ILSpy/Views/CreateListDialog.xaml.cs

```diff
@@ -31,5 +34,11 @@ private void OKButton_Click(object sender, RoutedEventArgs e)
 			get => ListNameBox.Text;
 			set => ListNameBox.Text = value;
 		}
+
+		protected override void OnSourceInitialized(EventArgs e)
+		{
+			base.OnSourceInitialized(e);
+			ThemeManager.Current.ApplyThemeForWindow(this);
```

Comment:
Code duplication: We don't need to repeat this for every window, (SPOM), it can be done once in the `WindowStyleManagerBehavior`; also other window related code better goes there.

Reply:
If the main window has a dark color, will the child window also have it? I have not tested this.

Reply:
`WindowStyleManagerBehavior` is attached to every window by style

Reply:
@tom-englert what should we remove from #2412 if anything?